PureBasicIDE/AddTools.pb

```diff
@@ -225,7 +225,14 @@ Procedure AddTools_ExecuteCurrent(Trigger, *Target.CompileTarget)
     Protected NewList EnvVars.s()
     
     AddTools_SetEnvVar(EnvVars(), "IDE", ProgramFilename())
-    AddTools_SetEnvVar(EnvVars(), "Compiler", *CurrentCompiler\Executable$) ; use *CurrentCompiler to have the real compiler path if multiple compiler are available
+    
+    Protected *TargetCompiler.Compiler
+    If *Target\CustomCompiler
+      *TargetCompiler = FindCompiler(*Target\CompilerVersion$)
```

Comment:
You need to check the return value of  `FindCompiler()`. It is 0 when no matching compiler is found. So there is potential for a crash here. I suggest switching to the `DefaultCompiler` if this is the case.

Reply:
I have added the change.